src/google.py

```diff
@@ -59,14 +60,32 @@ def login():
         
     except ValueError as ex:
```

Comment:
If the token is expired, does this exception handle it also? 🤔

Reply:
checks for token expiration has been included in the recent commits.